### PR TITLE
[Communication] - Identity and Common - Addressed final identity and common apiview comments

### PR DESCRIPTION
**Pull request by jbeauregardb**

### CODE REVIEW

**Review by minnieliu — APPROVED** [review]

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Release History
 
 ## 1.0.0-beta.5 (Unreleased)
+- Renamed `CommunicationTokenRefreshOptions.getRefreshProactively()` to `CommunicationTokenRefreshOptions.isRefreshProactively()`
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-common/CHANGELOG.md:4]: Add "Breaking Changes"

File: sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/MicrosoftTeamsUserIdentifier.java [hunk 2]
```diff
@@ -8,28 +8,14 @@
...
-public class MicrosoftTeamsUserIdentifier extends CommunicationIdentifier {
+public final class MicrosoftTeamsUserIdentifier extends CommunicationIdentifier {
 
     private final String userId;
     private final boolean isAnonymous;
     private CommunicationCloudEnvironment cloudEnvironment = CommunicationCloudEnvironment.PUBLIC;
 
     private String rawId;
 
-    /**
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/MicrosoftTeamsUserIdentifier.java:19]: Any reason this has to be removed?

File: sdk/communication/azure-communication-common/CHANGELOG.md (hunk 1 above)
Comment (minnieliu) [inline sdk/communication/azure-communication-common/CHANGELOG.md:4]: Do we need to update the date to 2021-03-09? I think it was mentioned in the standup

**Review by JianpingChen — CHANGES_REQUESTED** [review]

File: sdk/communication/azure-communication-common/CHANGELOG.md (hunk 1 above)
Comment (jbeauregardb) [reply sdk/communication/azure-communication-common/CHANGELOG.md:4]: Thank you Minnie! Changing it right now :)

File: sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/MicrosoftTeamsUserIdentifier.java (hunk 2 above)
Comment (jbeauregardb) [reply sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/MicrosoftTeamsUserIdentifier.java:19]: There's an apiview comment saying that this constructor isn't needed as users can just chain the cloudEnvironment using the set function we already have.

File: sdk/communication/azure-communication-common/CHANGELOG.md (hunk 1 above)
Comment (JianpingChen) [reply sdk/communication/azure-communication-common/CHANGELOG.md]: We normally do that as the last pr before release. The (Unreleased) tag will prevent the pipeline from releasing. Change it early is actually a risk. :-)

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,13 @@
 # Release History
 
+## 1.0.0-beta.6 (2021-03-09)
+- `CommunicationCloudEnvironment`, `CommunicationTokenRefreshOptions `, `CommunicationUserIdentifier`, `MicrosoftTeamsUserIdentifier`,
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-common/CHANGELOG.md:4]: Making classes final is actually a breaking change.

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0-beta.6 (2021-03-09)
+- Added a retryPolicy() chain method to the `CommunicationIdentityClientBuilder`
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:4]: "### Added" or "### New Features"

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
...
+- `CommunicationIdentityClient` had functions returning Void objects. This was fixed to make them void.
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:5]: It's a minor change. We can skip

**Review by JianpingChen — CHANGES_REQUESTED** [review]

**Review by JianpingChen — APPROVED** [review]

**Pull request merged**